doc/api/dgram.md

```diff
@@ -299,6 +299,62 @@ added: v0.6.9
 Sets or clears the `SO_BROADCAST` socket option.  When set to `true`, UDP
 packets may be sent to a local interface's broadcast address.
 
+### socket.setMulticastInterface(multicastInterface)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `multicastInterface` {String}
+
+Sets the default outgoing multicast interface of the socket to a chosen interface or back to system interface selection. The `multicastInterface` must be a valid string representation of an IP from the socket's family.
+
+For IPv4 sockets, this should be the IP configured for the desired physical interface. All packets sent to multicast on the socket will be sent on the interface determined by the most recent successful use of this call.
+
+For IPv6 sockets, `multicastInterface` should include a Scope ("IP%Scope") to indicate the interface as in the examples that follow. In IPv6, individual send calls can also use explicit Scope in addresses, so only packets sent to a multicast address without specifying an explicit Scope are affected by the most recent successful use of this call.
+ 
+#### Examples: IPv6 Outgoing Multicast Interface
+On most systems, where Scope format uses the interface name:
```

Comment:
It seems odd to me that there isn't a very good language neutral description of them to reference since every environment that runs on windows and one *nix needs the same documentation. Looking at libuv, the topic is skipped all together.

But I think it would make more sense to do a follow up patch to add scope_name to [os.networkInterfaces()](https://nodejs.org/api/os.html#os_os_networkinterfaces) and just link to whatever docs can be handled there.